tests/security/atsec/check_undocumented_security_programs.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'Check for undocumented security-relevant programs' test case of ATSec test suite
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#111671
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # The programs are defined by the FSP and corresponding man pages
```

Comment:
@Amrysliu 
Do you have a link to mention this in cc mode？

Reply:
No, it's mentioned in the test steps.